images/ci-operator-minimal/Dockerfile

```diff
@@ -1,8 +1,5 @@
 FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 
-RUN microdnf install -y git python3
```

Comment:
Isn't python needed anymore?

Reply:
This particular dockerfile is not getting used by any jobs. I am updating this image to test without python from an earlier discussion on thread between Brad and Danilo.

Reply:
Alright got it, this is `ci-operator-minimal`, we also have the fully fledged `ci-operator`.